.rubocop_todo.yml

```diff
@@ -0,0 +1,33 @@
+# This configuration was generated by
```

Comment:
I like the idea of the `.rubocop_todo.yml` but as I understand it, it's meant for things that want to be fixed at some point (that why the "TODO"). So I wonder, is it our case here?

Reply:
@santib that's a good point.

I like the idea as well, but it kinda force us to consider every violation as "tech debt". I think we should all be aligned on this since it will impact the whole company in some way.

IMO: Every violation should be considered as tech debt. If there are no plans to fix it, does the rule make sense? is it useful? Maybe it just needs to be disabled on the main config file.
I can't think of an scenario where we want to keep the offense and still consider that rule useful.

Reply:
I agree with what you said. In general terms think there are 2 kinds of exceptions:
1. Tech debt: things that should be fixed at some point
2. Directory level: some directories might need different rules, especially when using DSLs like RSpec or ActiveAdmin

For the rails_api_base I think we should only have the second kind of exceptions (which are not intended to be fixed) so projects start with a clean state